pkg/db/dto/git_provider.go

```diff
@@ -7,29 +7,42 @@ import (
 	"github.com/daytonaio/daytona/pkg/gitprovider"
 )
 
+type SigningMethod string
```

Comment:
No need to define a separate enum here. You can use `gitprovider.SigningMethod` below.

This will also eliminate the need for casting so `(*SigningMethod)(gitProvider.SigningMethod)` will become `gitProvider.SigningMethod` (this applies in a few places)

Reply:
done